### DISCUSSION TITLE
Tutorial Chinese Translation

### DISCUSSION THREAD

**Discussion by hulihutu9** [vitalik/django-ninja#1124] [General]
I want to translate the tutorial to Chinese, how could I do it?

**Comment by vitalik** [comment]
hi @hulihutu9 

inside a docs folder there is "docs" - copy it so it becomse docs/zn (f.e) and translate each file

keep in mind maintaining a language up to date is a hard task - do you think you can handle to time to time come apply new translations to new sections or changes ?

**Reply by hulihutu9** [reply]
I have not so much time , but I can translate the tutorial and guide in docs, so it can let more people learning Django-Ninja.
